articles/aks/security-bulletins/overview.md

```diff
@@ -22,6 +22,29 @@ These updates cover security information related to the following AKS components
 - Azure Kubernetes Service Node Image (AKS Node Image)
 - Azure Kubernetes Service Addons (AKS add-ons)
 
+---
+## AKS-2025-009 Important Security Update for Calico Users
+
+**Published Date**: June 25, 2025
+
+### Description
+This bulletin provides an update on the security patching model for Calico in Azure Kubernetes Service (AKS). AKS-managed Calico is now fully aligned with upstream [Calico releases](https://github.com/projectcalico/calico/releases). This means that AKS will no longer independently patch Calico images but will instead mirror upstream Calico builds directly. 
+
+As a result, CVEs affecting Calico will remain unpatched in AKS until a fix is available upstream. This change ensures consistency with upstream behavior and improves transparency in patch timelines.
+
+### References
+[Calico Release](https://github.com/projectcalico/calico/releases)
+
+### Affected Components
+
+#### [**AKS Cluster**](#tab/aks-cluster)
+
+**Affected Versions**
+- All AKS supported versions using AKS managed Calico
+
+**Resolutions**
+No immediate action is required. Customers are encouraged to monitor upstream Calico releases and the [AKS CVE Status Tracker](https://releases.aks.azure.com/webpage/index.html) for updates.
```

Comment:
Addthis? If this creates an unreasonable security burdon customers may remove calico by setting netowkr policy to none https://learn.microsoft.com/en-us/azure/aks/use-network-policies#uninstall-azure-network-policy-manager-or-calico